src/content/docs/snowflake/rest-api-endpoints.md

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
@HarshCasper we already have a doc for REST API here:
https://github.com/localstack/localstack-docs/blob/main/src/content/docs/snowflake/features/rest-api-endpoints.md 

Can you please consolidate and place in the right directory?